client/src/pages/Profile.js

```diff
@@ -125,6 +140,31 @@ export default function Profile({ authProps, setAuthProps }) {
     setEditingInfo(false);
   };
 
+  const delay = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+  const changeDietitian = async (err, data) => {
+    if (err) {
+      toast.error(err.message);
+    } else {
+      await axios
+        .put(
+          `http://localhost:8000/api/user/put?user_email=${authProps.user.user_email}`,
```

Comment:
this api link will have to be adjusted for the deployed heroku server api

Reply:
The adjusted api link for the deployed heroku link: https://nutrimental-server.herokuapp.com/api....